owncloudData/src/main/java/com/owncloud/android/data/folderbackup/db/FolderBackUpEntity.kt

```diff
@@ -31,12 +31,9 @@ data class FolderBackUpEntity(
     val uploadPath: String,
     val wifiOnly: Boolean,
     val chargingOnly: Boolean,
-    @ColumnInfo(name = folderBackUpEntityNameField) val name: String,
+    @ColumnInfo(name = "name")
+    val name: String,
```

Comment:
If the name we want to apply is the same as the entity attribute, we don't need the column info. Don't know why I introduced it initially 👀 
```suggestion
    val name: String,
```

Reply:
That's what I thought, but I didn't want to break it just in case 😬

Reply:
If this change broke something, we would be notified by a new DB scheme change. If it is not generated, go ahead 👍